nautobot/core/tests/runner.py

```diff
@@ -149,7 +158,7 @@ def setup_databases(self, **kwargs):
                                     verbosity=self.verbosity,
                                     keepdb=self.keepdb
                                     # Extra check added for Nautobot:
-                                    and not settings.TEST_USE_FACTORIES,
```

Comment:
The `TEST_USE_FACTORIES` check was added in #5130, but seems in my local tests to not actually be needed here - its presence here meant that we were never reusing the cloned database copies when running in parallel, which seems now to be an unnecessary precaution.

Reply:
This turned out to be necessary after all - see #6318.